Bottom: 2eb494d99a798d8027fb436887977340a27bc96c
Top:    fc96efd9ca533bb3c515c1f03319ead1d9f8b91a
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-08-07 11:30:00 -0700

Refresh of dbg-out

---

diff --git a/src/preload.c b/src/preload.c
index 55ee535..6a89d41 100644
--- a/src/preload.c
+++ b/src/preload.c
@@ -99,7 +99,7 @@ static int rq_size;
 static int sq_inline;
 static int fork_support;
 
-static FILE fout;
+static FILE *fout;
 
 enum fd_type {
 	fd_normal,
